hack/python-sdk/client-gen.sh

```diff
@@ -32,15 +32,23 @@ then
 fi
 
 echo "Generating Python SDK for KFServing ..."
-java -jar ${SWAGGER_CODEGEN_JAR} generate -i ${SWAGGER_CODEGEN_FILE} -l python -o ${SDK_OUTPUT_PATH} -c ${SWAGGER_CODEGEN_CONF}
+java -jar ${SWAGGER_CODEGEN_JAR} generate -i ${SWAGGER_CODEGEN_FILE} -g python -o ${SDK_OUTPUT_PATH} -c ${SWAGGER_CODEGEN_CONF}
 
 # revert following files since they are diveraged from generated ones
-git checkout python/kfserving/kfserving/rest.py
+#git checkout python/kfserving/kfserving/rest.py
```

Comment:
delete these two lines if they are not needed?

Reply:
I think that;s no need to checkout, will remove that.